### PR TITLE
Variable name cleanup

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: libbeat/outputs/outputs.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 type MothershipConfig struct {
-	Save_topology     bool
+	SaveTopology      bool
```
Comment (tsg) [inline libbeat/outputs/outputs.go]: Shouldn't this one get a yaml tag to "save_topology"?

File: libbeat/outputs/outputs.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ type MothershipConfig struct {
 	Path              string
 	Template          Template
 	Db                int
-	Db_topology       int
+	DbTopology        int
```
Comment (tsg) [inline libbeat/outputs/outputs.go]: Same on this one, needs "db_topology" from what i can see.

Comment (ruflin) [inline libbeat/outputs/outputs.go]: Fixed

File: libbeat/outputs/outputs.go (hunk 1 above)
Comment (ruflin) [inline libbeat/outputs/outputs.go]: Fixed

File: libbeat/outputs/outputs.go [hunk 3]
```diff
@@ -6,11 +6,11 @@ import (
 )
 
 type MothershipConfig struct {
-	Save_topology     bool
+	SaveTopology      bool `yaml:"save_topology"`
 	Host              string
 	Port              int
 	Hosts             []string
-	LoadBalance       *bool
+	LoadBalance       *bool `yaml:"loadbalance"`
```
Comment (ruflin) [inline libbeat/outputs/outputs.go]: @urso I added this change as I expected it to be called load_balance in the config, but it is loadbalance ...

**Pull request merged**